cypress/integration/HomeworkTests/calculator.spec.js

```diff
@@ -0,0 +1,155 @@
+/// <reference types="cypress" />
+
+describe('Asych', function () {
+    beforeEach(function () {
+        cy.visit('https://testsheepnz.github.io/BasicCalculator')
+
+    })
+
+    it('dropdown', function (){
+        cy.get('#selectBuild').select('1')
+        cy.get('#number1Field').type('3')
+        cy.get('#number2Field').type('3')
+        cy.get('#selectOperationDropdown').select('Multiply')
+        cy.get('#calculateButton').click()
+        cy.get('#numberAnswerField').should('have.value', '9')
+
+    })
+})
+
+
+describe('Asych', function () {
+    beforeEach(function () {
+        cy.visit('https://testsheepnz.github.io/BasicCalculator')
+
+    })
+
+    it.only('dropdown', function (){
```

Comment:
Matau rašai 'function ()', kas nėra neteisinga, bet dažnai pavyzdžiuose matysi arrow funkciją => 
https://stackoverflow.com/questions/24900875/whats-the-meaning-of-an-arrow-formed-from-equals-greater-than-in-javas
Čia gali paskaityti apie ją daugiau